src/main/scala/strawman/collection/Iterable.scala

```diff
@@ -81,17 +81,22 @@ trait IterableOps[+A] extends Any {
   /** A view representing the elements of this collection. */
   def view: View[A] = View.fromIterator(iterator())
 
-  /** Given a collection factory `fi` for collections of type constructor `C`,
-    *  convert this collection to one of type `C[A]`. Example uses:
+  /** Given a collection factory `fi`, convert this collection to the appropriate
+    * representation for the current element type `A`. Example uses:
     *
     *      xs.to(List)
     *      xs.to(ArrayBuffer)
+    *      xs.to(BitSet) // for xs: Iterable[Int]
     */
-  def to[C[X] <: Iterable[X]](fi: FromIterable[C]): C[A @uncheckedVariance] =
-  // variance seems sound because `to` could just as well have been added
-  // as a decorator. We should investigate this further to be sure.
+  def to[F <: TypeConstrainedFromIterable[A]](fi: F): fi.To[A @uncheckedVariance] =
     fi.fromIterable(coll)
```

Comment:
I don't think you need the `F` parameter here. 

    def to(fi: TypeConstrainedFromIterable[A]): fi.to[A]

should work, no?